src/components/atoms/infoCard/InfoCard.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../../foundations/sass/all";
+
+.info-card {
+  @include center;
+  gap: map-get($spacing, sm);
+  // background-color: $primary-color;
+  color: $white-floral;
+
```

Comment:
What are your thoughts on adding the `display: flex` property? Could you review the design? I think the info card doesn't take up the full width of the screen.

Reply:
I used flex just center the element, why it takes the full width, is there a problem ?